parser.y

```diff
@@ -414,6 +414,14 @@ import (
 	than		"THAN"
 	timeType	"TIME"
 	timestampType	"TIMESTAMP"
+	tokudbDefault "TOKUDB_DEFAULT"
```

Comment:
Should use `tab` to separate the two columns here.

Reply:
Fixed